SQL/New_patches/2024-25-06-CleanMriScanType.sql

```diff
@@ -0,0 +1,51 @@
+-- Rename foreign key fields for consistency
+
+ALTER TABLE `mri_scan_type`
+  RENAME COLUMN `ID` TO `MriScanTypeID`;
```

Comment:
I get the following error when trying to apply the patch:
```
$ mysql -A 'cecile_raisin_bread' < SQL/New_patches/2024-25-06-CleanMriScanType.sql 
ERROR 1064 (42000) at line 3: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'COLUMN `ID` TO `MriScanTypeID`' at line 2
```

Reply:
Instead of `RENAME`, you should probably use `MODIFY COLUMN` to rename the field.

Reply:
Oh interesting, which database are you using, it worked on mine so I thought it was standard :thinking:.

Reply:
Apparently this is what is used in my MCIN sandbox: 
```
mysql  Ver 8.0.39-0ubuntu0.20.04.1 for Linux on x86_64 ((Ubuntu))
```

Reply:
I copied #9270 and used the `CHANGE` syntax. The inconsistent but inconsequential types are copied from the existing schema.